python/pyproject.toml

```diff
@@ -32,6 +32,8 @@ pydantic = "^2"
 motor = "^3.3.2"
 defusedxml = "^0.7.1"
 pybars4 = "^0.9.13"
+jinja2 = "^3.1.3"
```

Comment:
Move these to the dev dependencies group below so they aren't dependencies of the whole project

Reply:
Sounds good! I'm not the one that added Jinja, so hopefully tests still pass!